index.html

```diff
@@ -181,7 +181,7 @@ <h2>Others about AntennaPod</h2>
 		<p class="centered">AntennaPod was created by Daniel Oeh and released under the MIT License.<br/>
 			            It is currently maintained by volunteers. Wanna help? Join us on <a href="https://www.github.com/antennapod/AntennaPod">GitHub</a>!</p>
                 <p class="centered">Website built using <a href="http://www.getbootstrap.com">Bootstrap, <a href="http://www.fontawesome.io/">FontAwesome</a> and the <a href="http://www.bootstrapzero.com/bootstrap-template/flatty">Flatty Template</a></p>
-		<p class="centered"><a href="https://twitter.com/antennapod"><i class="fa fa-twitter"></i></a> - <a href="https://groups.google.com/forum/#!forum/antennapod"><i class="fa fa-envelope"></i></a> - <a href="https://www.github.com/antennapod/AntennaPod"><i class="fa fa-github"></i></a> - <a href="https://en.wikipedia.org/wiki/AntennaPod"><i class="fa fa-wikipedia-w"></i></a> - <a href="https://www.transifex.com/antennapod/antennapod/"><i class="fa fa-language"></i></a><!-- - <a href="bitcoin:1DzvtuvdW8VhDsq9GUytMyALmsHeaHEKbg"><i class="fa fa-bitcoin"></i></a>--></p>
+		<p class="centered"><a href="https://twitter.com/antennapod"><i class="fa fa-twitter"></i></a> - <a href="https://forum.antennapod.org/"><i class="fa fa-envelope"></i></a> - <a href="https://www.github.com/antennapod/AntennaPod"><i class="fa fa-github"></i></a> - <a href="https://en.wikipedia.org/wiki/AntennaPod"><i class="fa fa-wikipedia-w"></i></a> - <a href="https://www.transifex.com/antennapod/antennapod/"><i class="fa fa-language"></i></a><!-- - <a href="bitcoin:1DzvtuvdW8VhDsq9GUytMyALmsHeaHEKbg"><i class="fa fa-bitcoin"></i></a>--></p>
```

Comment:
I'm so sorry to only get back to this now @Stimmenhotel! I think the enveloppe doesn't really cover the thing. What would you say of [speech bubbles](https://fontawesome.com/v4.7.0/icon/comments)?

Reply:
No problem. Bubbles would look better, you are right. I just wasn't sure how to change them. HTML/CSS is not my best language and I never used fonts awesome before.
(Also first commits to any github, but looks like it worked?)

Reply:
Well, there's a first for everything, and it all worked. :) Thanks!